scripts/redcap/purge_outdated.py

```diff
@@ -204,13 +203,15 @@ def main(api, args):
     marks = main(redcap_api, args)
 
     # Log dataframe by row here, one for exceeds, and another for precedes
-    log_dataframe_by_row(marks[marks['exceeds']], uid_template=UID_TEMPLATE, message="Entry form exceeds visit date.",
-        description="Listed is an entry form that exceeds the visit date by 120 days.")
-    log_dataframe_by_row(marks[marks['precedes']], uid_template=UID_TEMPLATE, message="Entry forms precedes visit date.",
-        description="Listed is an entry form that precedes the visit date.")
+    log_dataframe_by_row(marks[marks['exceeds']], uid_template=UID_TEMPLATE, 
+        message=f"Form exceeds visit date by {args.max_days_after_visit} days",
+        resolution="Contact site to determine if the date is incorrect and should be changed, if the form should be emptied, or if an exception should be set.")
+    log_dataframe_by_row(marks[marks['precedes']], uid_template=UID_TEMPLATE, 
+        message=f"Form precedes visit date by {args.max_days_after_visit} days",
```

Comment:
That's not the logic - "form precedes visit date by a non-zero number of days" would be more accurate, but "form precedes visit date" should be clear enough